### ISSUE TITLE
Screen reader navigation support for TreeView

### ISSUE THREAD

**Issue by banatiarpit97** [adobe/react-spectrum#3719]
# 🐛 Bug Report 

Navigating through screen reader does not work well with the TreeView items.

## 🤔 Expected Behavior

User should be able to to navigate through the complete list of TreeView items using screen reader(narrator) shortcuts.

## 😯 Current Behavior

In case of a long list inside TreeView, user is able to navigate only through the items visible on the screen at that moment. Any items that are hidden because of scroll position can not be reached using screen reader shortcuts.

## 🔦 Context

We have a long list of items present inside TreeView. Since TreeView component implement virtualised list which only render the items that are visible to the user, all others items are not present in DOM and hence screen reader can not reach those items.

## 🌍 Your Environment

Operating System - Windows(working fine on MAC)
react-spectrum - 2.37.0
Browser - all browsers

Turn on narrator and move focus on TreeView container, one can try to navigate through its children using arrow keys.

## 🧢 Your Company/Team

Adobe/Document Cloud

**Comment by snowystinger** [comment]
Please use slack for v2 support.
